libs/platform/core/src/services/page-meta/page-meta-resolver/page-meta-resolver.model.ts

```diff
@@ -0,0 +1,11 @@
+export interface ElementResolver {
+  title?: string;
+  [key: string]: string | undefined;
+}
+
+export const enum ResolverScore {
+  NotUsed = -1,
+  Fallback = 0,
+  Default = 1,
+  OverrideDefault = 2,
+}
```

Comment:
I'm not sure about these. Agreed that having a few standard numbers is great, but we'd need to be able to override the override.